Sources/Logging/Logging.swift

```diff
@@ -86,59 +148,176 @@ extension Logger {
             self.handler.logLevel = newValue
         }
     }
-
 }
 
 extension Logger {
+    /// Log a message passing with the `Logger.info` log level.
```

Comment:
This reads a bit weird... "passing with the ... log level"?

Maybe like this instead:

```
/// Log a message on the `Logger.Level.info` level
```

?

Reply:
The missing `Level` type is only here, not on the other levels.

Though the wording, whichever we use should be consistent then.